lib/flipper/ui/action.rb

```diff
@@ -202,16 +196,6 @@ def active?
         end
       end
 
-      # Public: Add a breadcrumb to the trail.
-      #
-      # text - The String text for the breadcrumb.
-      # href - The String href for the anchor tag (optional). If nil, breadcrumb
-      #        is assumed to be the end of the trail.
-      def breadcrumb(text, href = nil)
-        breadcrumb_href = href.nil? ? href : "#{script_name}#{href}"
-        @breadcrumbs << Breadcrumb.new(text, breadcrumb_href)
-      end
-
```

Comment:
This is technically a breaking change, but I'm not sure that this method would be used outside of this code.